### PR TITLE
Select summons

### PR DESCRIPTION
**Pull request by Anisetobegan**

### CODE REVIEW

File: Assets/Scripts/Character.cs [hunk 1]
```diff
@@ -27,15 +29,34 @@ enum State
 
     Vector3 lastPos;
 
+
+    private Camera cam;
+    public LayerMask clickable;
+    public LayerMask ground;
```
Comment (morathias) [inline Assets/Scripts/Character.cs]: Make LayerMask variables private serialized field

File: Assets/Scripts/Character.cs [hunk 2]
```diff
@@ -137,14 +158,76 @@ private void AddPerk(/*Perk perk*/)
...
 
-    private void GiveOrder(Summon selectedSummons)
+    private void GiveOrder()
     {
+        if (Input.GetMouseButtonDown(1) && selectedSummons.Count > 0)
+        {
+            RaycastHit hit;
+            Ray ray = cam.ScreenPointToRay(Input.mousePosition);
+
+            Debug.Log("giving order");
```
Comment (morathias) [inline Assets/Scripts/Character.cs]: Remove Debug

File: Assets/Scripts/Character.cs [hunk 3]
```diff
@@ -137,14 +158,76 @@ private void AddPerk(/*Perk perk*/)
...
 
+            if (Physics.Raycast(ray, out hit, Mathf.Infinity, ground))
+            {
+                foreach (Summon summon in selectedSummons)
+                {
+                    Debug.Log(hit.point);
```
Comment (morathias) [inline Assets/Scripts/Character.cs]: Here as well

**Review by morathias — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Character.cs (hunk 1 above)
Comment (Anisetobegan) [reply Assets/Scripts/Character.cs]: Done

File: Assets/Scripts/Character.cs (hunk 2 above)
Comment (Anisetobegan) [reply Assets/Scripts/Character.cs]: Done

File: Assets/Scripts/Character.cs (hunk 3 above)
Comment (Anisetobegan) [reply Assets/Scripts/Character.cs]: Done

**Review by morathias — APPROVED** [review]

**Pull request merged**